cylc/flow/network/__init__.py

```diff
@@ -46,19 +45,17 @@
 MSG_TIMEOUT = "TIMEOUT"
 
 
-def encode_(message):
+def JSON_stringify(message: object) -> str:
```

Comment:
It's usually convention to have functions and methods all lowercase in python, no?